docs/4.x/tech/installation-guide/system-requirements.adoc

```diff
@@ -44,6 +44,13 @@ CleanSpeak supports the following two databases.
 * MySQL 8 or newer
 * PostgreSQL 10 or newer
 
+=== Search
+
+CleanSpeak supports the following two search engine options:
+
+* OpenSearch 2.x
+* ElasticSearch >= 7.6.1
```

Comment:
@robotdan - Does FA say **less than/equal to 7.17** https://fusionauth.io/docs/get-started/download-and-install/system-requirements#elasticsearch <= 7.17 due to licensing? Should I say there here as well?

Reply:
It seems like the license change was 7.10 to 7.11 (https://www.elastic.co/pricing/faq/licensing) and, if you don't redistribute it (as of CS 4.x, we don't), it shouldn't affect people that "I build a SaaS application using Elasticsearch as the backend, how does this affect me"

Reply:
We haven't upgraded the version we are shipping in FusionAuth in part due to licensing changes. 

But the support statement (when it was written anyway) was just to say what we had tested up to. I think it also says other versions may work, but we may not have tested them. 

So for this support statement in CleanSpeak, i think `>= 7.6.1` is accurate. In practice - if a greater version doesn't work for some reason and someone opens a bug with us, we would want to fix it anyway. So in that sense it is supported even if we are not committing to testing every version.